rhea/cores/fifo/fifo_fast.py

```diff
@@ -97,7 +97,10 @@ def rtl_fifo():
 
         elif fbus.write and not fbus.read:
             fbus.empty.next = False
-            if not fbus.empty:
+            # because addr starts from 0,
+            # there should not be an increment
+            # the first time 
+            if (not fbus.empty) and (not fbus.full):
```

Comment:
Will remove them for now with the possibility of verifying SRL fifo and adding them later in a separate PR